unified-runtime/test/layers/tracing/codeloc.match

```diff
@@ -0,0 +1,5 @@
+RUN: %use-mock %xptienable XPTI_SUBSCRIBERS=%{shlibpre}test_collector%{shlibext} tracing-test-codeloc 2>&1 | FileCheck %s
+REQUIRES: filecheck
```

Comment:
I'm still missing the value of this `REQUIRES`

Reply:
IMO, yes. If someone is forced to install `lit` anyway, requiring `FileCheck` as part of it seems very reasonable.

Reply:
I agree we should require `FileCheck` for all `lit` based testing even if some tests don't use it.